cli/src/deno.rs

```diff
@@ -109,13 +109,14 @@ impl ExtensionsModuleLoader {
         Ok(fs::read_to_string(path).await?)
     }
 
-    async fn load_from_deno_std(path: &Url, domain: &str) -> Result<String> {
-        if domain == "deno.land" {
+    async fn load_from_deno_std(path: &Url) -> Result<String> {
+        if let Some(Host::Domain("deno.land")) = path.host() {
```

Comment:
I *feel* like match would be the way to go here? though this certainly works.

Reply:
I'm not gonna die on this hill, fine with both. But `_ =>` would work perfectly fine too of course.

I just generally feel like `if let` + `else` is somewhat rare, but if clippy doesn't warn on it I don't mind it either.